eli5/base.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+import attr
+from typing import Dict, List, Tuple, Union
+
+
+@attr.s
+class Explanation(object):
```

Comment:
Right, docstrings and some comment would be definitely nice here, let me add them tomorrow :)